docker/ciip_postgres_backup/Dockerfile

```diff
@@ -0,0 +1,10 @@
+COPY root /
```

Comment:
You need a `FROM debian` or something, otherwise you're building on top of a blank kernel (I think). @wenzowski any base image preference?